commit 647715605fcf046ee42a62997588a61772ff94b3
Author: Yiannis Marangos <yiannis.marangos@gmail.com>
Date:   Mon Oct 16 23:50:25 2017 +0800

    UBUNTU: SAUCE: Input: synaptics - Lenovo X1 Carbon 5 should use SMBUS/RMI
    
    BugLink: http://bugs.launchpad.net/bugs/1723986
    
    Link: https://patchwork.kernel.org/patch/9989549/
    
    Signed-off-by: Yiannis Marangos <yiannis.marangos@gmail.com>
    Signed-off-by: Aaron Ma <aaron.ma@canonical.com>
    Acked-by: Hui Wang <hui.wang@canonical.com>
    Acked-by: AceLan Kao <acelan.kao@canonical.com>
    Signed-off-by: Seth Forshee <seth.forshee@canonical.com>

diff --git a/drivers/input/mouse/synaptics.c b/drivers/input/mouse/synaptics.c
index ee5466a374bf..b3c683a84d3f 100644
--- a/drivers/input/mouse/synaptics.c
+++ b/drivers/input/mouse/synaptics.c
@@ -172,6 +172,7 @@ static const char * const smbus_pnp_ids[] = {
 	"LEN0048", /* X1 Carbon 3 */
 	"LEN0046", /* X250 */
 	"LEN004a", /* W541 */
+	"LEN0073", /* X1 Carbon 5 */
 	"LEN200f", /* T450s */
 	NULL
 };
